README.md

```diff
@@ -18,11 +18,15 @@ zip_extract::extract(Cursor::new(archive), &target_dir, true)?;
 
 
 ## Features
-All features are passed through to `zip` and `flate2`. They are:
 
+All features are the same as the features of the `zip` crate:
+
+- `aes-crypto`: Support for AES encryption via the the `zip` crate
 - `deflate`: Support for the Deflate algorithm (`miniz_oxide` rust-backend)
 - `deflate-miniz`: ^ dito (`miniz` C-backend)
 - `deflate-zlib`: ^ dito (`zlib` C-backend)
+- `unreserved`: Support for the `unreserved` feature of the `zip` crate
 - `bzip2`: Support for .bzip2 archives via the `bzip2` crate
-
-The default is `bzip2` and `deflate`.
+- `time`: Support for the `time` crate for the `zip` crate
+- `zstd`: Support for the `zstd` crate for the `zip` crate
+- `default`: enables `"aes-crypto", "bzip2", "deflate", "time", "zstd"`
```

Comment:
this library doesn't use `time` or `aes-crypto`, do we really want to enable them by default?

Reply:
I am mirroring the upstream defaults here. Doesn't zip-extract use these indirectly?

Reply:
You're right, encrypted archives are implicitly supported, so removing that is technically a breaking change.

The `time` features however - we just copy around bytes, we never check the modification time. So we can safely disable that one.